crates/typos-cli/src/config.rs

```diff
@@ -455,15 +485,16 @@ impl DictConfig {
     }
 }
 
-fn find_project_file(dir: &std::path::Path, names: &[&str]) -> Option<std::path::PathBuf> {
+fn find_project_files(dir: &std::path::Path, names: &[&str]) -> Vec<std::path::PathBuf> {
     let mut file_path = dir.join("placeholder");
+    let mut found_files = vec![];
     for name in names {
         file_path.set_file_name(name);
         if file_path.exists() {
-            return Some(file_path);
+            found_files.push(file_path.clone());
```

Comment:
nit: we could make the return type `impl Iterator<Item=PathBuf>` if we make the function body
```rust
names.iter().map(|name| dir.join(name)).filter(|path| path.exists())
```